.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+omit =
+    */genetic_testing/assay_target/primer_design.py
```

Comment:
Could you add a note here why this is omitted for future reference?

Reply:
Sure, I have added a small note.